src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -351,6 +356,7 @@ private T createOrUpdate(Request req, Response res) {
                 }
                 // Run pre-create hook and use updated object (with potentially modified values) in create operation.
                 persistence.create(preCreateHook(newEntity, req));
+                postCreateHook(newEntity, req);
```

Comment:
I'm guessing that we want the input to `postCreateHook` be the return value of `preCreateHook` because the first hook will sometimes modify `newEntity`:

```java
T updatedEntity = preCreateHook(newEntity, req);
persistence.create(updatedEntity);
postCreateHook(updatedEntity, req);
```

Reply:
Updated in 6948fab3bf5ead8b87a4b47676a38fe9a6e1691f